openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -523,6 +532,17 @@ public void postProcessing() {
 			coreEdgeFilter.add(new WheelchairCoreEdgeFilter(gs));
 		}
 
+		/* Maximum Speed Filter */
+		if (routingProfileCategory !=0 & (encodingManager.hasEncoder("heavyvehicle") || encodingManager.hasEncoder("car-ors")) ) {
```

Comment:
Why not simply use `routingProfileCategory == RoutingProfileCategory.DRIVING` as conditional?

Reply:
We talked with Hendrik and decided to do it only for these vehicles. but I can do it in general.